src/org/labkey/test/tests/FieldEditorRowSelectionActionTest.java

```diff
@@ -83,6 +88,19 @@ public void testListActions() throws Exception
 
         checker().verifyTrue("Exported Fields are not same as UI", uiFields.equals(exportedFields));
 
+        log("Verifying the toggle between summary view and detail view list domain");
```

Comment:
The section here, and the block at line 127, and at 200, have a lot of similarities and the differences they do have look like they could be adjusted via function arguments, if they were modularized into a function `testSummaryViewActions` or something.

Would it be good style to group these three chunks into one function, or in tests is it preferable to have the code be more explicit like this?

Reply:
I thought about it, it was mostly assert statements and not a lot of action having. Decided to keep them in the test.

Let me know if you feel strongly to move.